netmaster/master/endpointGroup.go

```diff
@@ -67,10 +73,14 @@ func CreateEndpointGroup(tenantName, networkName, groupName string) error {
 	// params for docker network
 	if GetClusterMode() == "docker" {
```

Comment:
not part of your change, but the naked strings need to be enumerated. if you can fix it, please do. otherwise, file an issue